src/app/board/question/major/page.tsx

```
@@ -100,10 +101,32 @@ export default function MajorQuestionPage() {
+  };
+
+  const handleRemoveFilter = async (filterType: "sortType" | "chaetaekStatus" | "tag", value?: string) => {
+    const newFiltering: Partial<QuestionCommand> = { ...currentFiltering };
+
+    if (filterType === "sortType") {
+      delete newFiltering.sortType;
+    } else if (filterType === "chaetaekStatus") {
+      delete newFiltering.chaetaekStatus;
+    } else if (filterType === "tag" && value) {
+      newFiltering.questionPresetTags = (newFiltering.questionPresetTags || []).filter(tag => tag !== value);
+      if (newFiltering.questionPresetTags.length === 0) delete newFiltering.questionPresetTags;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)